devices/usbc-cdc/usbphy-imxrt117x.c

```diff
@@ -0,0 +1,154 @@
+/*
+ * Phoenix-RTOS
+ *
+ * Operating system loader
+ *
+ * i.MX RT117x USB physical layer controller
+ *
+ * Copyright 2021 Phoenix Systems
+ * Author: Gerard Swiderski
+ *
+ * This file is part of Phoenix-RTOS.
+ *
+ * %LICENSE%
+ */
+
+#include "client.h"
+#include "usbphy.h"
+
+
+/* Size of memory pool aligned to USB_BUFFER_SIZE, used by USB descriptors */
+#define USB_POOL_SIZE ((ENDPOINTS_DIR_NB * ENDPOINTS_NUMBER + 1) * USB_BUFFER_SIZE)
+
+
+static struct {
+	unsigned char pool[USB_POOL_SIZE] __attribute__((aligned(USB_BUFFER_SIZE)));
```

Comment:
`u8` is shorter

Name `pool` looks better so to be consistent you can change this variable also in the other phy layers.